internal/test/wmcb_test.go

```diff
@@ -0,0 +1,224 @@
+	// in the list of services
+	// TODO: Parse the output of the `Get-Service sshd, ssh-agent` on the Windows node to check if the windows nodes
+	// has those services present
+	time.Sleep(time.Minute)
+	if err := framework.configureOpenSSHServer(); err != nil {
+		log.Fatalf("failed configuring OpenSSHServer on the Windows VM created with error: %v", err)
+	}
+	if err := framework.createRemoteDir(); err != nil {
+		log.Fatalf("failed to create remote dir with error: %v", err)
+	}
+	if err := framework.getSSHClient(); err != nil {
+		log.Fatalf("failed getting ssh client for the Windows VM created with error: %v", err)
+	}
+}
+
+// createWindowsVM spins up the Windows VM in the given cloud provider and gives us the credentials to access the
+// windows VM created
+func (f *testFramework) createWindowsVM() error {
+	kubeconfig := os.Getenv("KUBECONFIG")
+	awsCredentials := os.Getenv("AWS_SHARED_CREDENTIALS_FILE")
+	dir := os.Getenv("ARTIFACT_DIR")
+	privateKeyPath := os.Getenv("KUBE_SSH_KEY_PATH")
+	// Use Windows 2019 server image with containers in us-east1 zone for CI testing.
+	// TODO: Move to environment variable that can be fetched from the cloud provider
+	imageID := "ami-0b8d82dea356226d3"
+	instanceType := "m4.large"
+	sshKey := "libra"
+	cloud, err := cloudprovider.CloudProviderFactory(kubeconfig, awsCredentials, "default", dir,
+		imageID, instanceType, sshKey, privateKeyPath)
+	if err != nil {
+		return fmt.Errorf("instantiante cloud provider %v", err)
+	}
+	f.cloudProvider = cloud
+	credentials, err := cloud.CreateWindowsVM()
+	if err != nil {
+		return fmt.Errorf("create Windows VM: %v", err)
+	}
+	f.credentials = credentials
+	return nil
+}
+
+// setupWinRMClient sets up the winrm client to be used while accessing Windows node
+func (f *testFramework) setupWinRMClient() error {
+	host := f.credentials.GetIPAddress()
+	password := f.credentials.GetPassword()
+
+	endpoint := winrm.NewEndpoint(host, winRMPort, true, true, nil, nil, nil, 0)
+	winrmClient, err := winrm.NewClient(endpoint, user, password)
+	if err != nil {
+		return fmt.Errorf("failed to set up winrm client with error: %v", err)
+	}
+	f.winrmClient = winrmClient
+	return nil
+}
+
+// configureOpenSSHServer configures the OpenSSH server using WinRM client installed on the Windows VM.
+// The OpenSSH server is installed as part of WNI tool's CreateVM method.
+func (f *testFramework) configureOpenSSHServer() error {
+	// This dependency is needed for the subsequent module installation we're doing. This version of NuGet
+	// needed for OpenSSH server 0.0.1
+	installDependentPackages := "Install-PackageProvider -Name NuGet -MinimumVersion 2.8.5.201 -Force"
+	if _, err := f.winrmClient.Run(remotePowerShellCmdPrefix+installDependentPackages,
+		os.Stdout, os.Stderr); err != nil {
+		return fmt.Errorf("failed to install dependent packages for OpenSSH server with error %v", err)
```

Comment:
Shouldn't we be logging `os.Stderr` too? Otherwise debugging this from CI will be difficult. Comment applies to all places.